### PR TITLE
homepage cards - remaining style issues

### PR DESCRIPTION
**Pull request by jsandland**
**Fixes following issues|Adds following features:**
- Fix left spacing on discover cards and gen-ai on smaller screens
- Ensure card navigation moves to bottom right of GenAI and remove linear gradient on left to right navigation
- Ensure responsive gen-ai card sizing

**Resolves:** [MWPW-159393](https://jira.corp.adobe.com/browse/MWPW-159393)

**Pages to check for regression and performance:**
- https://cards-home-page--express--adobecom.hlx.page/drafts/integration/simplified-homepage?martech=off

### CODE REVIEW

File: express/blocks/discover-cards/discover-cards.css [hunk 1]
```diff
@@ -71,7 +71,16 @@ main .section .discover-cards {
...
+.discover-cards .cards-container.gallery {
+    scroll-padding-left: 20px;
+}  
+.discover-cards .cards-container .card:first-child {
+    margin-left: 20px;
+}
+
+
+
+@media (min-width: 400px) {
```
Comment (kennethlum) [inline express/blocks/discover-cards/discover-cards.css:83]: min-width of 400px, so here is there a special handling needed for extra small phone like iPhone SE vs iPhone 15 Pro Max (which is 430)?  (by the way, I found that even iPhone 12 Pro Max is 375, the same as iPhone SE)

Comment (jsandland) [reply express/blocks/discover-cards/discover-cards.css:83]: everything above 400 will get these rules - do we need to handle 375? I'm using the standard breakpoints I see in the express repo.

Comment (kennethlum) [reply express/blocks/discover-cards/discover-cards.css:83]: oh I mean it seems you are distinguishing between 375px and 430px... so I wonder if there is something about it like you have to do something special for 375, but for 400 and up you can relax it

File: express/blocks/gen-ai-cards/gen-ai-cards.css [hunk 2]
```diff
@@ -249,3 +259,30 @@
...
+  .gen-ai-cards.homepage .carousel-element.card .media-wrapper {
+    width: 325px;
+  }
+  .gen-ai-cards.homepage .carousel-element.card {
+    width: 357px;
+    height: 396px;
+  }
+  .gen-ai-cards.homepage .card .text-wrapper {
+    text-align: left;
+    margin: 20px 20px 12px 20px;
```
Comment (kennethlum) [inline express/blocks/gen-ai-cards/gen-ai-cards.css:284]: nit: the last `20px` can be omitted

**Review by fullcolorcoder — APPROVED** [review]

**Pull request merged**